compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/App.kt

```diff
@@ -54,6 +54,7 @@ val MainScreen = Screen.Selection(
     Screen.Example("GraphicsLayerSettings") { GraphicsLayerSettings() },
     Screen.Example("Blending") { Blending() },
     Screen.Example("FontRasterization") { FontRasterization() },
+    Screen.Example("InteropOrder") { InteropOrder() },
```

Comment:
This demo isn't always work correctly (at least on Windows):

(reproducible 1 in 3 cases):

- Press Red - it is hidden, but the Green is over Blue:
<img width="262" alt="image" src="https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/f045e89c-56b7-4861-9935-6ae782df91bd">

or

- Press Blue - it is hidden, but the Red is over Green:
<img width="292" alt="image" src="https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/0585724b-bcb0-44ad-98da-1d1093f46481">

Reply:
Fixed by force revalidation of Swing container